airflow/models/crypto.py

```diff
@@ -85,7 +87,7 @@ def get_fernet():
                 for fernet_part in fernet_key.split(',')
             ])
             _fernet.is_encrypted = True
-    except (ValueError, TypeError) as ve:
-        raise AirflowException("Could not create Fernet object: {}".format(ve))
+    except (ValueError, TypeError) as value_error:
+        raise AirflowException("Could not create Fernet object: {}".format(value_error))
```

Comment:
```suggestion
        raise AirflowException(f"Could not create Fernet object: {value_error}"))
```
Let us use the occasion to introduce f-string 😄